zomes/test/src/lib.rs

```diff
@@ -195,6 +196,61 @@ fn emit_signal_from_sibling_cell(payload: SiblingEmitPayload) -> ExternResult<()
     Ok(())
 }
 
+#[hdk_extern]
+fn create_cap_grant_for_private_function(_: ()) -> ExternResult<CapSecret> {
+    let cap_secret = generate_cap_secret()?;
+
+    let mut functions: GrantedFunctions = BTreeSet::new();
+    functions.insert((zome_info()?.name, "private_function".into()));
+
+    create_cap_grant(CapGrantEntry {
+        tag: "".into(),
+        access: cap_secret.into(),
+        functions,
+    })?;
+
+    Ok(cap_secret)
+}
+
+#[hdk_extern]
+fn private_function(_: ()) -> ExternResult<String> {
```

Comment:
@zo-el - just want to check that it's ok to just return a String here. I picked up something along the way that I should wrap a value like this in a struct (eg like TestObj) in this file. But it seems to work fine just returning a String. Any issue with this?